server/src/main/java/org/apache/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -499,6 +503,23 @@ public boolean doMonitor(ServiceEmitter emitter)
       final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder();
       MonitorUtils.addDimensionsToBuilder(builder, dimensions);
       emitter.emit(builder.build("jetty/numOpenConnections", ACTIVE_CONNECTIONS.get()));
+      ThreadPool jettyServerThreadPool = JETTY_SERVER_THREAD_POOL.get();
+      emitter.emit(builder.build("jetty/threadPoolNumTotalThreads", jettyServerThreadPool.getThreads()));
+      emitter.emit(builder.build("jetty/threadPoolNumIdleThreads", jettyServerThreadPool.getIdleThreads()));
+      emitter.emit(builder.build("jetty/threadPoolIsLowOnThreads", jettyServerThreadPool.isLowOnThreads() ? 1 : 0));
+      if (jettyServerThreadPool instanceof ThreadPool.SizedThreadPool) {
+        emitter.emit(builder.build("jetty/threadPoolMinThreads",
+                                   ((ThreadPool.SizedThreadPool) jettyServerThreadPool).getMinThreads()));
+        emitter.emit(builder.build("jetty/threadPoolMaxThreads",
+                                   ((ThreadPool.SizedThreadPool) jettyServerThreadPool).getMaxThreads()));
+      }
+      if (jettyServerThreadPool instanceof QueuedThreadPool) {
```

Comment:
I don't think this check is necessary. `jettyServerThreadPool` could be declared as `QueuedThreadPool`

Reply:
Thanks! Addressed.